lang/lang.go

```diff
@@ -0,0 +1,60 @@
+package lang
+
+// New returns a new and initialized Lang.
+func New(language string) Lang {
+	l := Lang{language: language}
+
+	// Add all lanaguages here, the keys should be named according to BCP47.
+	l.m = map[string]Term{
+		"en": Term{
+			Footnotes:    "Footnotes",
+			Bibliography: "Bibliography",
+			Index:        "Index",
+		},
+		"nl": Term{
+			Footnotes:    "Voetnoten",
+			Bibliography: "Bibliografie",
+			Index:        "Index",
+		},
+	}
```

Comment:
Thank you for quick implementation. I'd request:
```suggestion
		"ja": Term{
			Footnotes:    "脚注",
			Bibliography: "参考文献",
			Index:        "索引",
		},
		"zh-cn": Term{
			Footnotes:    "注记",
			Bibliography: "参考文献",
			Index:        "索引",
		},
		"zh-tw": Term{
			Footnotes:    "註記",
			Bibliography: "參考文獻",
			Index:        "索引",
		},
	}
```